assets/stylesheets/components/_toast.scss

```diff
@@ -0,0 +1,44 @@
+#toast-container {
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  z-index: 6;
```

Comment:
@tcmacdonald: So, yes, `6` is context-specific. However, I feel like we should have some default index here that can be overridden locally. Otherwise, it's likely to fall behind some element at some point. And this is the one element we _know_ should always be on top, correct?

I'm not a huge fan of the `99999999999999` index, but this may be a case for at least a few nines.

Thoughts?